src/EORegistryCoordinator.sol

```diff
@@ -56,9 +56,9 @@ contract EORegistryCoordinator is
     IEOChainManager public EOChainManager;
```

Comment:
storage vars should not start with a capital. I would not even add the prefix EO for var names just for types.
recommend renaming to chainManager instead.

Reply:
BTW slither also throws a warning about this.

Reply:
changed